### ISSUE TITLE
[Sourcing][Accessibility]:  fd-vertical-navigation accessibility issues

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9608]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
1) when focus is on "deliveries" the screenreader doesnt read UI control type "list item"
2) even for nested list items , the UI control type is not read by screenreader
3) when the parent node is expanded and the focus is on the parent node, the focus is appearing for entire list , rather it should be moved to parent node and the levels have to be read out by screen reader

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
core / platform: 0.39.2

#### Is there anything else we should know?

Please find the screenshots for issue 1, 2 and 3 

Issue 1
<img width="1792" alt="Screenshot 2023-03-30 at 5 00 07 PM" src="https://user-images.githubusercontent.com/88182358/228823463-61f13566-8319-43a3-b728-12bd40869bc5.png">

Issue 2
<img width="1792" alt="Screenshot 2023-03-30 at 5 00 29 PM" src="https://user-images.githubusercontent.com/88182358/228823505-26973435-f0ec-41b2-9b67-6286b786d59c.png">

Issue 3
<img width="1792" alt="Screenshot 2023-03-30 at 5 00 38 PM" src="https://user-images.githubusercontent.com/88182358/228823548-9b602382-2256-4a8c-abc2-b883cb6d737e.png">

**Comment by mikerodonnell89** [comment]
We are missing the correct roles for side nav and vertical nav, but the screenreader specifications state that this is all they should need (specifically the table needs role="tree" and the nodes role="treeitem", not list/listitem respectively). UI5 is using aria-label with invisible text to force the screenreader to announce "list item", I'm not sure that is truly the desired behavior so I've reached out to our accessibility contacts about it
